MT_Test_Startup.ahk

```diff
@@ -0,0 +1,100 @@
+﻿#Include <definitions>
```

Comment:
Issue #7 

Tested on my Local pc. Not yet tested in CI. logs are attached below.
[MT_Local_Test_Startup_Logs.TXT](https://github.com/NWLdragonite/MT_Automator/files/43589/MT_Local_Test_Startup_Logs.TXT)